references/cheat-sheet.md

```diff
@@ -0,0 +1,22 @@
+# Cheat Sheet
+
```

Comment:
I'd love to see a sentence or two as a summary/intro for what these are.

Reply:
considering that a "cheat sheet" that you can print to your cubicle wall is an Industry Best Practise :tm: - I don't see a reason to add any lead-in just for the sake of adding text - kube agrees, see https://kubernetes.io/docs/reference/kubectl/cheatsheet/

Not saying that it shouldn't have, but this comes down to writers personality. I would only add a passive aggressive "here are some handy tricks that you can print to your cubicles wall" text.

As you're leading this guide effort (?), you should write that appropriate lead-in text that is just right for the tone you wan't to have in kontena docs.  I can't know what that tone should be (and I don't see a reason, but that's just me) - I just know that a catch-all place for one-liners/tricks that are not guide worthy should exist.

Reply:
Fair enough. I might add something in a different PR